src/pages/global-specific-settings.js

```diff
@@ -108,15 +108,15 @@ class GlobalSpecificSettings extends React.Component {
                         handleClose={this.props.specificSettingDialogClose}
                         dataTitle={this.props.specificSettingDataTitle}
                         dataTitleOptions={this.props.specificSettingOptions}
-                        titleValue={this.props.states.specificSettingName}
+                        titleValue={this.props.statesSpecific.name}
```

Comment:
Is this is a typo?  Why is it called `statesSpecific` here?

Reply:
I'm changing the property name `statesSpecific` to `SpecificSetting` in all components.